packages/expo-print/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### 🐛 Bug fixes
 
 - Fix calculating the number of pages when printing HTML on iOS. ([#13633](https://github.com/expo/expo/pull/13633) by [@dsokal](https://github.com/dsokal))
+- Add markupFormatter option in printToFileAsync (default option can't handle html page breaks). ([#13799](https://github.com/expo/expo/pull/13799) by [@mstach60161](https://github.com/mstach60161))
```

Comment:
Usually we use backticks for words that refer to something in the code 😉 I would also consider moving this from bug fixes to new features 🤔 It would be a bug fix if upgrading the package fixes it without changing application code, but in that case the user need to switch over to the new option. Besides, the new option is a new feature anyway 😄 

```suggestion
- Add `markupFormatterIOS` option in `printToFileAsync` on iOS (default option can't handle html page breaks). ([#13799](https://github.com/expo/expo/pull/13799) by [@mstach60161](https://github.com/mstach60161))
```